test/use_case/statistics/StatisticsTest.java

```diff
@@ -0,0 +1,47 @@
+package use_case.statistics;
+
+import data_access.InMemoryGameDataAccessObject;
+import entity.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+public class StatisticsTest {
+    /**
+     * Initialize each test to have an existing game with the current round already set and a data access object.
+     */
+    private final int INITIAL_LIVES = 10;
+    private StatisticsDataAccessInterface gameDataAccessObject;
+    private Game game;
+    @Before
+    public void init() {
```

Comment:
For the testing, I have a few suggestions. 
1) Hardcode a few games with different data before the tests. This way the tests will be more realistic since our application is likely to have multiple games in memory played. 
2) Right now you have tested your methods for the stats well. We also want to add testing for the interactor/outpudata, etc. To do this you want to create dummy input data, anonymous presenter where you put your assert statements, and a dummy interactor which you call. For example see the code below (this test does not work but should give you an outline of how to structure your tests to get complete code coverage):
```
 @Test
    public void commonDifficultyTest() {
        StatisticsOutputBoundary statsPresenter = new StatisticsOutputBoundary() {
            @Override
            public void prepareView(StatisticsOutputData outputData) {
                // have your assert statements here verifying info about the stats and your output data
            }
        }
        StatisticsInputBoundary interactor = new StatisticsInteractor(gameDataAccessObject,  statsPresenter);
        interactor.execute();
    }
```